apps/docs/docs/state/recipes/_category_.json

```diff
@@ -1,5 +1,5 @@
 {
   "label": "Recipes",
-  "position": 3,
+  "position": 80,
```

Comment:
Why are you changing the sidebar position to x10 number?
Did you mean 8?

Reply:
This is the 3rd last item.